client/src/index.jsx

```diff
@@ -0,0 +1,6 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+
+import App from './components/App.jsx';
+
+ReactDOM.render(<App />, document.getElementById('app'));
```

Comment:
are you getting warnings in your console? check the [docs](https://reactjs.org/blog/2022/03/08/react-18-upgrade-guide.html) for how to update this to use the latest version's syntax